extension/src/lib/highContrast.js

```diff
@@ -0,0 +1,87 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+// This component makes use of css/highContrast.css to perform the adaptations
+// of the web content, and this is done through chrome.tabs.executeScript.
+//
+fluid.defaults("gpii.chrome.highContrast", {
+    gradeNames: ["fluid.modelComponent"],
+    scriptTemplate: "document.documentElement.setAttribute('data-gpii-hc','__THEME__');" +
```

Comment:
Ok, here's the thing.
I can inject this code as functions inside a content script (in fact I did it on the past), but it will be injected always, which is something I'd prefer not to do. Also, if the extension is enabled after the user has started using the browser, the content script won't be loaded into every window/tab, so we have to go over the tabs manually and make sure the content script is injected.
These are the reasons why I'm performing a [programmatic code injection](https://developer.chrome.com/extensions/content_scripts#pi). I've put the disableScript inside a file and injecting it when necessary, but the scriptTemplate needs some adjustments prior to be injected (replace __THEME__ with the current highcontrast theme). Besides storing the scriptTemplate as a file and reading it as a string .... do you think of any other ideas/alternatives?